pkg/templater/random.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright © 2022 - 2024 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package templater
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/rand/v2"
```

Comment:
Would it be possible to use `crypto/rand` instead? Maybe a bit overkill, but doesn't harm I think.

Reply:
In this case rand/v2 would be enough: it introduced dynamic seeding and reworked the random number generator algs wrt rand/v1.  But well, better safe than sorry, and if we always stick to crypto/rand we will not "forget" to use it where it could be really needed in the project.

Reply:
Well, honestly I'm not that happy with crypto/rand here: it is a bit cumbersome generating simple random INTs and we don't really need "secure" numbers. Mainly the "big".Int thing requirement is annoying (while of course it makes a lot of sense in security contexts).
Here it is just a minimal rand usage, so doesn't really matter what library we use... sticking with crypto/rand.
But if we will ever need more random operations in the projects with no requirements from a security PoV let's stick with math/rand/v2.
So, long story short, changed my mind about "always" sticking with crypto/rand in the project if security is not involved 😆